### PR TITLE
DOCSP-34594 clarify use case for deleting network containers

### PR DESCRIPTION
**Pull request by kanchana-mongodb**
## Proposed changes

_Jira ticket:_ DOCSP-34594

Closes #[issue number]

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

File: docs/atlascli/command/atlas-networking-containers-delete.txt [hunk 1]
```diff
@@ -40,7 +40,7 @@ Arguments
    * - containerId
      - string
      - true
-     - Unique 24-hexadecimal digit string that identifies the network container that you want to remove.
+     - Unique 24-hexadecimal digit string that identifies the network container that you want to remove
```
Comment (kyuan-mongodb) [inline docs/atlascli/command/atlas-networking-containers-delete.txt:43]: Question: Shouldn't there be a period at the end? 
Applies to the ,go file too with the same sentence.

**Review by kyuan-mongodb — APPROVED** [review]
LGTM % question!

File: docs/atlascli/command/atlas-networking-containers-delete.txt (hunk 1 above)
Comment (kanchana-mongodb) [reply docs/atlascli/command/atlas-networking-containers-delete.txt:43]: It is rendering correctly (with period) in the [external docs](https://www.mongodb.com/docs/atlas/cli/stable/command/atlas-networking-containers-delete/).

File: internal/cli/atlas/networking/containers/delete.go [hunk 2]
```diff
@@ -53,11 +53,11 @@ func DeleteBuilder() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "delete <containerId>",
 		Aliases: []string{"rm"},
-		Short:   "Remove the specified network peering container from your project.",
+		Short:   "Remove the specified network peering container from your project before creating any clusters. Don't run this command if you have clusters in your project.",
 		Long:    fmt.Sprintf(usage.RequiredRole, "Project Owner"),
 		Args:    require.ExactArgs(1),
 		Annotations: map[string]string{
-			"containerIdDesc": "Unique 24-hexadecimal digit string that identifies the network container that you want to remove.",
+			"containerIdDesc": "Unique 24-hexadecimal digit string that identifies the network container that you want to remove",
```
Comment (gssbzn) [inline internal/cli/atlas/networking/containers/delete.go:60]: @kanchana-mongodb I think what @kyuan-mongodb means is you deleted this period, was that intentional? could we rollback this change?
```suggestion
			"containerIdDesc": "Unique 24-hexadecimal digit string that identifies the network container that you want to remove.",
```

Comment (kanchana-mongodb) [reply internal/cli/atlas/networking/containers/delete.go]: Thanks @gssbzn!

**Comment by gssbzn** [comment]
needs a run on `make gen-docs`

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**